src/Features/Core/Portable/Workspace/BackgroundCompiler.cs

```diff
@@ -42,6 +43,25 @@ public BackgroundCompiler(Workspace workspace)
             }
         }
 
+        public void Dispose()
+        {
+            if (_workspace != null)
+            {
+                this.CancelBuild(releasePreviousCompilations: true);
+
+                var editorWorkspace = _workspace as Workspace;
```

Comment:
Heh. Copied it from the constructor without noticing that. I will fix in both places.